modules/network-graph-20-view.adoc

```diff
@@ -0,0 +1,69 @@
+* The legend provides information about the symbols in use and their meaning. The legend shows explanatory text for symbols representing namespaces, deployments, and connections on the network graph.
+* You can select additional display options from the drop-down list to display icons such as the network policy status badge, active external traffic badge, and port and protocol labels for edge connections.
+* When {product-title-short} detects changes in network traffic, such as nodes joining or leaving, the network graph displays a notification showing the number of updates available. To avoid interrupting your focus, the graph is not updated automatically. Click the notification to update the graph.
+[discrete]
+== External entities and connections in Network Graph (2.0 preview)
+
+The network graph view shows network connections between managed clusters and external sources.
+In addition, {product-title-short} automatically discovers and highlights public Classless Inter-Domain Routing (CIDR) address blocks, such as Google Cloud, AWS, Azure, Oracle Cloud, and Cloudflare.
+Using this information, you can identify deployments with active external connections and determine if they are making or receiving unauthorized connections from outside your network.
+
+By default, the external connections point to a common *External Entities* box and different CIDR address blocks in the *Network Graph* view.
+However, you can choose not to show auto-discovered CIDR blocks by clicking *Manage CIDR blocks* and deselecting *Auto-discovered CIDR blocks*.
+
+##This section below needs review - is all of it still applicable to Network Graph version 2.0?##
+
+{product-title-short} includes IP ranges for the following cloud providers:
+
+* Google Cloud
+* AWS
+* Azure
+* Oracle Cloud
+* Cloudflare
+
+{product-title-short} fetches and updates the cloud providers' IP ranges every 7 days.
+If you are using offline mode, you can update these ranges by installing new support packages.
+
+.Network graph example
+
+The following image provides an example of the network graph. In this example, based on the options that the user has chosen, the graph depicts deployments and traffic flows between them. It also uses a red badge to indicate deployments that are missing policies and therefore allowing all network traffic.
+
+image::network-graph-20-overview-updated.png[Overview of the Network graph showing deployments and connections]
+
+When you click an item in the graph, the rearranged side panel with collapsible sections presents deployment and namespace information. The side panel switches between Deployment mode and Namespace mode based on item in the graph that you have selected. The D or NS label next to the item name in the header (in this example, "postgres,") indicates if it is a deployment or a namespace. The following example illustrates deployment mode.
```

Comment:
We have the following nodes that are selectable: Deployment, Namespace, External Entities, CIDR Blocks, and External groups (which just contain External Entities and CIDR Blocks).